packages/interactive-messages/src/http-handler.ts

```diff
@@ -121,7 +119,8 @@ export function createHTTPHandler(adapter) {
 
           const dispatchResult = adapter.dispatch(body);
 
-          if (dispatchResult) {
+          if (dispatchResult !== undefined) {
+            // tslint:disable-next-line no-floating-promises
```

Comment:
this seems dangerous. why would there be a floating promise? if we should be catching this and doing something with it, let's maybe mark that with a TODO?

Reply:
I can't say I really know *what* there would be to do after responding & I don't think its feasible to try and route this back to the dev 😬 so it looks like a TODO is the current choice.